gci/src/main/java/com/danielfireman/gci/Sampler.java

```diff
@@ -0,0 +1,49 @@
+package com.danielfireman.gci;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Sampler {
```

Comment:
Add a comment?

Reply:
You mean like a documentation?

Reply:
Yes. I meant [javadoc](http://www.oracle.com/technetwork/java/javase/tech/index-137868.html).